run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# Run original test harness
+/odh-operator-test-harness.test
+
+# Set up variables for ods-ci htpasswd provider
+export HTPASSWD_IDP_NAME='htpasswd'
+export HTPASSWD_USER="$(pwgen 32 1)"
+export HTPASSWD_PASSWORD="$(pwgen 32 1)"
+# CLUSTER_ID should be passed in via the harness, if not attempt to set it
+if [ x${CLUSTER_ID} == x ]; then
+  export CLUSTER_ID="$(oc get clusterversion -o jsonpath='{.items[].spec.clusterID}')"
+fi
+
+ocm login --token "${ocm-token-refresh}"
```

Comment:
Is that the actual variable name we need to use (ocm-token-refresh)?  I think it's problematic when passing env to the container (the dashes seem to be an issue).
Also, this command fails in the container even after I worked around the "dash problem":  Error: Can't save config file: can't create directory /.config/ocm: mkdir /.config: permission denied

Reply:
Interesting, I'm not sure, that was the value I was given but was never able to test it

Also, interesting seems I need to change the workdir. I had figured it would create it under $HOME xD